pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[project]
+name = "git-big-picture"
+description = "Git — the big picture"
+readme = "README.rst"
+license = "GPL-3.0-or-later"
+license-files = ["COPYING"]
+authors = [
+    { name = "Sebastian Pipping" },
+    { name = "Julius Plenz" },
+    { name = "Valentin Haenel" },
+]
+requires-python = ">=3.10"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
+    "Programming Language :: Python :: 3 :: Only",
+]
+dynamic = ["version"]
+
+[project.urls]
+Homepage = "https://github.com/git-big-picture/git-big-picture"
+
+[project.optional-dependencies]
+# Keep in sync with test_requirements.txt
+tests = ["coverage", "cram", "parameterized", "pytest"]
+
+[project.scripts]
+git-big-picture = "git_big_picture._main:main"
+
+[tool.setuptools.data-files]
+"share/man/man1" = ["git-big-picture.1"]
+
+[tool.setuptools.dynamic]
+version = { attr = "git_big_picture.__version__" }
+
+[tool.setuptools.packages.find]
+include = ["git_big_picture*"]
+
+[tool.ruff]
+indent-width = 4
+line-length = 99
+target-version = "py310"
+
+[tool.ruff.lint]
+select = [
+    "E",  # pycodestyle
+    "F",  # Pyflakes + flake8
+    "I",  # isort
+    "UP", # pyupgrade
+    "W",  # pycodestyle
+]
+ignore = [
+    "UP031", # Use format specifiers instead of percent format; TODO drop
+]
+
+[build-system]
+requires = ["setuptools>=77"]
```

Comment:
Out of curiosity (a question, not a blocker): How did you come to ">=77"?

Reply:
Good question, [v77](https://setuptools.pypa.io/en/latest/history.html#v77-0-0) was the first version to support the new [PEP 639](https://peps.python.org/pep-0639/#project-source-metadata) form of `license` + `license-files` metadata which this uses

Reply:
@branchvincent good to know, thanks :+1: